build/basil.js

```diff
@@ -217,8 +217,10 @@
 				// retrieve last updated cookie first
 				for (var i = cookies.length - 1, cookie; i >= 0; i--) {
 					cookie = cookies[i].replace(/^\s*/, '');
-					if (cookie.indexOf(key + '=') === 0)
-						return cookie.substring(key.length + 1, cookie.length);
+					if (cookie.indexOf(key + '=') === 0){
```

Comment:
Why not just having done

```javascript
if (cookie.indexOf(key + '=') === 0)
   return decodeURIComponent (cookie.substring(key.length + 1, cookie.length)); 
```

Would have spared a ligne, some brackets (that breaks CS actually with no space ;) )